core/state-machine/Cargo.toml

```diff
@@ -19,3 +19,13 @@ num-traits = "0.2"
 
 [dev-dependencies]
 hex-literal = "0.2.0"
+criterion = "0.2"
+rand = "0.6"
+
+[[bench]]
+name = "bench"
+harness = false
+
+[features]
+default = ["bench"]
+bench = []
```

Comment:
Can you leave a comment on this feature? Why would one enable this?

Reply:
yes was for bench, I cannot find a good way to activate a function for bench only (here 'bench' as a default feature is clearly wrong). Since it was only to make a function public, I am simply switching the function to public for now ('set_storage').